src/MyTested.AspNetCore.Mvc.TempData/Builders/Contracts/Data/IWithoutTempDataBuilder.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace MyTested.AspNetCore.Mvc.Builders.Contracts.Data
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Used for building <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+    /// </summary>
+    public interface IWithoutTempDataBuilder
+    {
+        /// <summary>
+        /// Remove temp data entry by providing its key to the built <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+        /// </summary>
+        /// <param name="key">Key of the temp data entry.</param>
+        /// <returns>The same <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.</returns>
+        IAndWithoutTempDataBuilder WithoutEntry(string key);
+
+        /// <summary>
+        /// Remove temp data entries by providing their keys to the built <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.
+        /// </summary>
+        /// <param name="entriesKeys">Keys of the temp data entries to be deleted.</param>
+        /// <returns>The same <see cref="Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionary"/>.</returns>
+        IAndWithoutTempDataBuilder WithoutEntries(IEnumerable<string> entriesKeys);
```

Comment:
All collection methods in the project have two overloads - one with IEnumerable<type> and one with params type[]. Add on every `Without` method with a collection - an overload, which accepts params. Search the project for params and you will find examples. Also - no need for the parameter to be called `entriesKeys`. Only `keys` is enough to be consistent with the method above.